restapi.go

```diff
@@ -2874,6 +2874,56 @@ func (s *Session) FollowupMessageDelete(appID string, interaction *Interaction,
 	return s.WebhookMessageDelete(appID, interaction.Token, messageID)
 }
 
+// ------------------------------------------------------------------------------------------------
+// Functions specific to stage instances
+// ------------------------------------------------------------------------------------------------
+
+// StageInstanceCreate will creates a new Stage instance associated to a Stage channel.
+// Returns that Stage instance created
+func (s *Session) StageInstanceCreate(data *StageInstanceParams) (si *StageInstance, err error) {
+	body, err := s.RequestWithBucketID("POST", EndpointStageInstances, data, EndpointStageInstances)
+	if err != nil {
+		return
+	}
+
+	err = unmarshal(body, &si)
+	return
+}
+
+// StageInstance will retrieve a Stage instance by ID of the Stage channel.
+// channelID : The ID of the Stage channel
+func (s *Session) StageInstance(channelID string) (si *StageInstance, err error) {
+	body, err := s.RequestWithBucketID("GET", EndpointStageInstance(channelID), nil, EndpointStageInstance(channelID))
+	if err != nil {
+		return
+	}
+
+	err = unmarshal(body, &si)
+	return
+}
+
+// StageInstanceEdit will edit a Stage instance by ID of the Stage channel.
+// channelID : The ID of the Stage channel
+// topic : The new topic of the Stage instance (1-120 characters). Set empty to keep the current topic.
+// privacyLevel : The new privacy level of the Stage instance. Set nil to keep the current privacy level.
+func (s *Session) StageInstanceEdit(channelID string, data *StageInstanceParams) (si *StageInstance, err error) {
```

Comment:
Actually, since we can edit only `topic`, I think it would make sense to just allow user to pass it directly as a functional param.

Reply:
We can edit the privacyLevel to `Public` in some rare case, feature are currently rolling out 

CF: https://support.discord.com/hc/en-us/articles/1500005513722-Stage-Channels-FAQ#h_01F68TX3MH2175AYXFKYJVG4WK